### ISSUE TITLE
Product subcategory wrog logic

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2449]
#### Issue

Here is videoscreen: http://screencast.com/t/J04LZltYr

I created fist the Category 1.0 and marked it as Parent

![cat1](https://cloud.githubusercontent.com/assets/5773777/19525507/786786b8-9621-11e6-9b11-f653b468df02.png)

After it i created in subpanel "Sub categories" the new one via quick edit window. Now you can see created subcategory 1.1

![cat2](https://cloud.githubusercontent.com/assets/5773777/19525576/b2eb6138-9621-11e6-971e-7b0a148aeb8e.png)

Now i clicked on subcategory 1.1 and in detailview i saw again the "Sub category" subpanel... and i created again the new one via quickedit window with name 1.1.1.

![cat3](https://cloud.githubusercontent.com/assets/5773777/19526260/267e412c-9624-11e6-8adc-ee6e4ad52b8a.png)

If you can see now the Subcategory 1.1 has own cubcategory but on detailview you can see that the 1.1 is not the parent category....

Now my question is.... Can have any category only one parent category ? or each subcategory can create self subcategory via supcategory supbanel
#### Expected Behavior

If category will not checked as parent category will have no possibility to create new record vie subpanel "Subcategories"
#### Possible Fix

You can create loop, that from any subcategory you can create other subcategory
#### Steps to Reproduce

1. Create parent category
2. Via subpanel create subcategory
3. Click on subcategory and go to detailview 
4. Now you create via subpanel other subcategory
#### Your Environment

- SuiteCRM Version used: 7.7.5 Official demo
- Browser name and version: Chrome 53.x m 32 bit
- Operating System and version: Win 10 Home

**Comment by shogunpol** [comment]
@Mausino , the difference between this two functionalities, is that:
- Is parent category - is place when you can select record as a parent cathegory (for example when is your first record),
- Parent category is place when you select already existing parent,
All Product categories and products work in "tree manner", so this is not good idea, and do not see any reason to create "loop" when two categories are parents to each other. 

So this functionality is work this way.
I suggest close issue, but if you think the functionality suppose to be changed we can label this issue as a suggestion,
Thanks for your contribution.

**Comment by Mausino** [comment]
@shogunpol  i understand your opinion, but on listview you are lost with structure of categories, because you don't know which is assigned to other.... you can see their relationship only in editview.. 

You can label it as suggestion, because if  "tree manner" will working as we need, the user need know the way as for exaple:  Garden -> Accessories --> Hammers  on listview or new view simmilary to Giant view where you can see this tree manner relationship (associations).... 

and also is problem with pop up windows, becase there is list of records (categories) but how you can find right one  for your product if you don't know all categories ?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Product subcategory wrog logic](https://trello.com/c/GThJsWG3/62-product-subcategory-wrog-logic)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
